### PR TITLE
Add digitalmarketplace team quotes

### PR DESCRIPTION
**Pull request by domoscargin**
As we want these to post in a different channel from our git notices, we've created a new team and selected the right channel.

### CODE REVIEW

**Review by lfdebrux — APPROVED** [review]
Thank you for doing this @domoscargin

**Review by bjgill — APPROVED** [review]
Nice. :shipit:

**Review by gidsg — APPROVED** [review]
LGTM 📇 👍

File: config/alphagov.yml [hunk 1]
```diff
@@ -95,6 +95,36 @@ digitalmarketplace:
...
+    - "We feel comfortable asking and answering questions from each other - no matter how basic"
+    - "We help each other grow and learn"
+    - "We are supportive and look out for each other"
+    - "We value each other's roles and contributions"
+    - "We are inclusive and considerate to others"
+    - "We communicate in an inclusive and effective manner"
+    - "We ensure meetings are focused with a clear purpose and relevant people, and keep to time"
+    - "We have meetings that are effective, short and have outcomes"
+    - "We respect each other's working styles and patterns"
+    - "We have a good work/life balance"
```
Comment (NoraGDS) [inline config/alphagov.yml]:
```suggestion
    - "We have a good life-work balance"
```

**Pull request merged**